tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -32,6 +32,12 @@ export const ooc_notes: Feature<string> = {
   component: FeatureTextInput,
 };
 
+export const headshot: Feature<string> = {
+  name: "Headshot",
+  description: "Add an image to your character, visible on close examination. Requires it be formatted properly.",
+  component: FeatureShortTextInput,
+};
+
```

Comment:
I'm not sure that really fits in the lines of "Species features", would make more sense in another file, probably.

Reply:
Done